### ISSUE TITLE
Draft Layout of Site

### ISSUE THREAD

**Issue by willoughbylabs** [erikamanning/1up#1]
- full page with navbar
- content displays in the center

**Comment by willoughbylabs** [comment]
@erikamanning Thank you for suggesting Figma, it is incredible. I like it better than Adobe XD, especially now that XD doesn't support my OS (but Figma's desktop app does!). I pulled some colors from the palettes you suggested. I'm not 100% sold on my choices but had to pick something! 

From what I've read, starting with a mobile-first design and expanding to desktop is recommended so that's what's mocked up first. Haven't built out all the pages, but Navbar works to navigate a bit. Please leave comments/thoughts/goodies here! Once we like a starting point, it should be easier to build out the portions working off the same design.

https://www.figma.com/proto/6rO5WM7lEnFclKqda0TORj/One-Up?node-id=1%3A2&viewport=91%2C354%2C0.6343467831611633&scaling=scale-down

**Comment by erikamanning** [comment]
I love the colors you picked in the Figma design!

**Comment by erikamanning** [comment]
I agree maybe it could be improved somewhat- but I can't think how and I'm definitely happy with it for now we can always tweak it later unless you have an idea! :)
